lib/utils/calculateAmount.js

```diff
@@ -0,0 +1,8 @@
+import { get } from 'lodash';
+
+// eslint-disable-next-line import/prefer-default-export
+export const calculateAmount = (fund, price) => {
+  const fundDistributionPercentage = get(fund, 'percentage', 0);
+
+  return ((fundDistributionPercentage / 100) * price).toFixed(2);
```

Comment:
that's actually wrong, so it's better to fix altogether. `getAmountWithCurrency` should be used, but you need to pass there local and currency. So technically, it's better to use component `AmountWithCurrencyField` in places you display any money.

Reply:
I agree with you, but this function only for calculate value for display. And i need to get amount for each fund. I think I need to rename this function and variable to make it more clear

Reply:
`.toFixed(2)` this part shouldn't be here

Reply:
I removed it. You may be reviewed old changes.